doc/userDocs/usersGuide.md

```diff
@@ -1356,4 +1403,79 @@ As a result, when using node labels it is recommended that you label all of your
 worker nodes with either `wvrole=driver` or `wvrole=sut`.  This will give you the 
 best control over pod placement.
 
+## Running on Specific Cluster Types<a name="specific"></a>
+
+### Running on Minikube
+
+This section discusses requirements for running Weathervane on a laptop or workstation using a 
+Kubernetes cluster created with [Minikube](https://minikube.sigs.k8s.io/docs/start/).
+
+#### Cluster Requirements
+
+In order to run Weathervane with a single Micro instance, you must create your Minikube 
+cluster with at least 3 cpus and 8GB of memory.  In addition, the default-storageclass 
+addon must be enabled (this is the default).  The micro instance will also use 15.25GB 
+of disk space.
+
+The command to start Minikube with 3 cpus and 8Gb is :
+* `minikube start --cpus=3 --memory=8g`
+
+Note that even with this cluster configuration, Minikube may have trouble running Weathervane 
+depending on how much memory is configured on the laptop and what else is 
+running.
+
+#### Weathervane Configuration
+
```

Comment:
To run on Minikube, the user should still follow the Quickstart guide, correct? And the setup diverges when it comes to configuring Weathervane. For completeness, maybe add something like:
"To run Weathervane on Minikube, follow the Quickstart Setup steps Obtain Weathervane from GitHub and Create the Weathervane Docker Images. Then follow the instructions below to create the Weathervane Configuration file when running on Minikube."

Reply:
Good idea.  I will add this.

Reply:
I actually added this in an overview section so that we don't have to repeat it for evey cluster type we describe here.